code/__DEFINES/reagents.dm

```diff
@@ -184,12 +184,6 @@
 #define GIN 			"gin"
 #define ABSINTHE 			"absinthe"
 #define RUM 			"rum"
-#define TEQUILA 			"tequila"
-#define VERMOUTH 			"vermouth"
-#define WINE 			"wine"
-#define COGNAC 			"cognac"
-#define HOOCH 			"hooch"
-#define ALE 			"ale"
```

Comment:
How odd. HOWEVER, most (all?) deadrum children, such as Classic Martini, Whiskey Cola, Vodka Martini, Screwdriver Cocktail, etc., have no "non-deadrum" equivalent.

So you definitely CAN get deadrum children in game.